felix/fv/infrastructure/felix.go

```diff
@@ -162,6 +163,12 @@ func RunFelix(infra DatastoreInfra, id int, options TopologyOptions) *Felix {
 	if fvBin == "" {
 		fvBin = fmt.Sprintf("bin/calico-felix-%s", arch)
 	}
+
+	if cwLogDir == "" {
+		var wDir, err = os.Getwd()
+		Expect(err).NotTo(HaveOccurred())
+		cwLogDir = filepath.Join(wDir, "/cwlogs")
```

Comment:
Could you clarify what the `cwLogDir` directory is used for and why we need this for the debugging? Is it related to logs generated through Goland?

Reply:
`cwLogDir is not related specifically to the Goland`. When you run tests via Makefile this variable is being set here https://github.com/skoryk-oleksandr/calico/blob/53cc83b4b6f4b01c796bd1b75773ed301bda10da/felix/fv/run-batches#L26
`export FV_CWLOGDIR=`pwd`/cwlogs` which will be `/{path to the project root}/felix/fv/cwlogs`
So basically I added logic that checks if this var is not set it use default value.

Reply:
Okay, I see. So you're basically providing a failsafe (default) value in case `FV_CWLOGDIR` is empty.